index.js

```diff
@@ -41,7 +42,12 @@ function Store (dbName, options) {
   var api = merge(
     db.hoodieSync({remote: remote, emitter: emitter}),
     db.hoodieApi({emitter: emitter}),
-    { hasLocalChanges: hasLocalChanges.bind(db) }
+    { 
+      hasLocalChanges: hasLocalChanges.bind(db),
+      push: syncWrapper.bind(db.hoodieSync({remote: remote, emitter: emitter}), 'push'),
+      pull: syncWrapper.bind(db.hoodieSync({remote: remote, emitter: emitter}), 'pull'),
+      sync: syncWrapper.bind(db.hoodieSync({remote: remote, emitter: emitter}), 'sync')
```

Comment:
can we initialise `db.hoodieSync({remote: remote, emitter: emitter})` only once, and then pass the instance to the 3 lines above?